src/i18n/fr_FR.ts

```diff
@@ -70,736 +70,442 @@ export default {
         version: {
             title: 'Mauvaise version de LSS Manager',
             text:
-                'Cher utilisateur, malheureusement vous n'avez pas la dernière version de LSS Manager. La dernière version est {version} et vous avez la {current}. Merci de recharger le jeu en vidant le cache (Ctrl + F5 ou command + R sur Apple), cela devrait régler le problème. Si le problème persiste, merci de le signaler à l\'équipe ! Si vous vous utilisez une mauvaise version nous ne pouvons garantir le plein fonctionnement de LSS-Manager.',
+                'Cher utilisateur, malheureusement vous n\'avez pas la dernière version de LSS Manager. La dernière version est {version} et vous avez la {current}. Merci de recharger le jeu en vidant le cache (Ctrl + F5 ou command + R sur Apple), cela devrait régler le problème. Si le problème persiste, merci de le signaler à l\'équipe ! Si vous vous utilisez une mauvaise version nous ne pouvons garantir le plein fonctionnement de LSS-Manager.',
             close: 'Fermer ce message et recharger le jeu (recommandé)',
             abort: 'Fermer ce message sans recharger le jeu',
         },
     },
     globalSettings: {
-        name: 'General Settings',
+        name: 'Paramètres généraux',
         labelInMenu: {
-            title: 'Label instead of icon in menu',
+            title: 'Titre au lieu d\'une icone dans le menu',
             description:
-                'Displays a simple label in the navigation bar instead of the LSSM logo',
+                'Un simple titre dans la barre de navigation au lieu du logo LSSM',
         },
     },
     vehicles: [

```

Comment:
Vehicles need to be changed in 
vehicles: **{**
0: {
and so for every id:  
id: {
caption: '',
color: '',